lms/services/aes.py

```diff
@@ -0,0 +1,20 @@
+from typing import Union
+
+from Cryptodome import Random
+from Cryptodome.Cipher import AES
+
+
+def decrypt(secret, iv, encrypted):
```

Comment:
This might not be estrictly neccesasry as we could use `jwt` to serialize the private keys for us.

Still I think this is nicer than having it on `application_instance`. 

Not using an  `AESService` injecting the secret as using it like that from a model it's awkward